### ISSUE TITLE
AsciiDoc

### ISSUE THREAD

**Issue by amelillo** [retext-project/pymarkups#19]
Hello,
I was wondering why if pymarkups documentation states support for **AsciiDoc** and the markups directory shows **asciidoc.py**, ReText _doesn't list AsciiDoc as one of the natively supported markup languages_.
Is there something I'm missing?
It would be great if I could edit&preview AsciiDoc with ReText as I do with Markdown.

**Comment by mitya57** [comment]
AsciiDoc support just has not been released yet.

You can help testing it by installing pymarkups from git, e.g. using this command:
```
pip install --upgrade git+https://github.com/retext-project/pymarkups.git#egg=Markups
```

Then it should be automatically picked up by ReText.

**Comment by amelillo** [comment]
Thanks.
I did it, with success, but still ReText 8.0 though stating to be using PyMarkups 3.1.3 doesn't list AsciiDoc as an available markup language.
![Schermata del 2023-01-11 22-40-13](https://user-images.githubusercontent.com/27341818/211924130-13fd424c-6fdb-4096-aa45-54f72ec764ff.png)

**Comment by amelillo** [comment]
Here it is. As I wrote, I used the command above to install pymarkups and ReTexs still doesn't show AsciiDoc as an available markup language.
![Schermata 2023-01-14 18:06:59](https://user-images.githubusercontent.com/27341818/212485809-8f9179d4-a800-4f37-be92-d894bc8f3572.png)

**Comment by amelillo** [comment]
I must be stupid, as I simply didn't install asciidoc. A quick pip and it works perfectly. Thanks.
Strange that when I installed pymarkups via pip it didn't lament any unsatisfied dependency.
Not so strange as they are *optional* deps, though.
Thank you again.

**Comment by mitya57** [comment]
And I have just released Markups 4.0.0 which includes AsciiDocMarkup. So for future just `pip install Markups[asciidoc]` should be enough.
